modules/ROOT/pages/authentication-authorization/index.adoc

```diff
@@ -45,11 +45,17 @@ A plugin option for building custom integrations.
 It is recommended that this option is used as part of a custom delivery as negotiated with link:https://neo4j.com/professional-services/[Neo4j Professional Services].
 For more information, see link:{neo4j-docs-base-uri}/java-reference/{page-version}/extending-neo4j/security-plugins#extending-neo4j-security-plugins[Java Reference -> Authentication and authorization plugins].
 
-
 *Kerberos authentication and single sign-on*::
 In addition to LDAP, native, and custom providers, Neo4j supports Kerberos for authentication and single sign-on.
 Kerberos support is provided via the link:{neo4j-docs-base-uri}/kerberos-add-on/current/[Neo4j Kerberos Add-On].
 
+*Mixed-mode authentication
+Neo4j also supports mixed-mode authentication that allows you to use multiple authentication providers in your database setup.
+For more information and examples, see xref:authentication-authorization/ldap-integration.adoc#auth-ldap-configure-provider[Set Neo4j to use LDAP] and xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-sso[Configure Neo4j to use OpenID Connect].
```

Comment:
so we don't need here referenced to the configuration settings, do we?

Reply:
I don't think so. Here is just a list of all supported providers with links to more information. We don't need to explain how to configure them. If we decide to do that, we need to rewrite the page and add that information for all of them.